applications/plugins/SofaMatrix/SofaMatrix.md

```diff
@@ -1,3 +1,60 @@
 # SofaMatrix
 
-SofaMatrix is a SOFA plugin containing components related to linear system matrices.
\ No newline at end of file
+SofaMatrix is a SOFA plugin containing components related to linear system matrices.
+
+## Components
+
+The plugin contains the following components that can be placed into a SOFA scene:
+
+### GlobalSystemMatrixExporter
+
+GlobalSystemMatrixExporter is a component to export the matrix of a linear system.
```

Comment:
Could you add where is the matrix exported by default?  and if it is possible to change the export path?

Reply:
Very good remark. Thanks